pysrc/bytewax/connectors/sql.py

```diff
@@ -0,0 +1,108 @@
+"""Connectors for [SQL](https://en.wikipedia.org/wiki/SQL).
```

Comment:
If this connector is ok to be in bytewax, I suggest to update docs:
https://github.com/bytewax/bytewax/blob/main/docs/articles/getting-started/ins_and_outs.md